src/store/factory/client.js

```diff
@@ -194,12 +198,22 @@ function createBSCClient (asset, network, mnemonic) {
   return createEthereumClient(asset, bnbNetwork, rpcApi, scraperApi, EthereumRpcFeeProvider, mnemonic)
 }
 
+function createPolygonClient (asset, network, mnemonic) {
+  const isTestnet = network === 'testnet'
+  const polygonNetwork = AssetNetworks.POLYGON[network]
+  const rpcApi = isTestnet ? 'https://rpc-mumbai.maticvigil.com/' : 'https://rpc-mainnet.maticvigil.com/'
+  const scraperApi = isTestnet ? 'https://liquality.io/polygon-testnet-api' : 'https://liquality.io/polygon-mainnet-api'
+
+  return createEthereumClient(asset, polygonNetwork, rpcApi, scraperApi, EthereumRpcFeeProvider, mnemonic)
```

Comment:
something is not right here. The `EthereumRpcFeeProvider` part has changed in dev but i don't see that reflected here

can you make sure your branch is upto speed?

Reply:
Updated and all changes from dev are merged into the current branch.